### PR TITLE
Skip 0 replica optimization

### PR DESCRIPTION
**Pull request by matzew**
see if we are still better w/ default to `replicas: 1` for the kafka channel deployment

### CODE REVIEW

File: knative-operator/deploy/resources/knativekafka/1-channel-consolidated.yaml [hunk 1]
```diff
@@ -728,7 +728,7 @@ metadata:
 spec:
   # this deployment is going to be scaled up by the
   # controller when the very first KafkaChannel is created
```
Comment (skonto) [inline knative-operator/deploy/resources/knativekafka/1-channel-consolidated.yaml:730]: Should we remove this comment since default is 1?

Comment (matzew) [reply knative-operator/deploy/resources/knativekafka/1-channel-consolidated.yaml:730]: yeah, we can patch remove the comments too

**Comment by matzew** [comment]
@skonto I've changed the patch, to also remove the comments!  Thanks for the feedback!

**Comment by markusthoemmes** [comment]
/retest

Infra.

**Review by aliok — APPROVED** [review]
/lgtm
/approve

Let's see how it goes.

**Pull request merged**